velox/docs/functions/spark/math.rst

```diff
@@ -214,6 +214,14 @@ Mathematical Functions
 
     Returns the modulus (remainder) of ``n`` divided by ``m``. Corresponds to Spark's operator ``%``.
 
+.. spark:function:: rint(x) -> double
+
+    Returns the double value that is closest in value to the argument and is 
+    equal to a mathematical integer.
+    If ``x`` is +/-Inf, it's returned unmodified. If ``x`` is Nan, Nan is returned. ::
```

Comment:
nit: Returns 'x' if x is a positive or negative infinity or a NaN.

BTW, would you double check that this behavior matches Spark?

Reply:
![image](https://github.com/facebookincubator/velox/assets/10524738/94b958c5-0b17-4c58-9fbd-aff8fa01679f)
Thanks.